docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,202 @@
+
+## Conversion Template Collection
+The convert data operation is based on the [FHIR Converter Project](https://github.com/microsoft/FHIR-Converter/tree/dotliquid).
+To make a data conversion call, you need a template collection to define the mappings between different data formats.
+
+We have a built-in template collection (hl7v2 to FHIR R4), which is a copy from the [FHIR Converter project](https://github.com/microsoft/FHIR-Converter/releases/tag/v3.0). Currently the version of templates is ```v3.0``` and we will periodically update to the latest stable release in the future.
+
+To convert with the built-in template collection, you can directly use the built-in template collection reference ```microsofthealth/fhirconverter:default``` in the request payload.
+
+If you want to refine or customize the output from the built-in template collection, you can also build your own template collections.
+To convert with custom template collection, you should follow the following steps:
+1. Get a copy of the [default template collection](https://github.com/microsoft/FHIR-Converter/releases/tag/v3.0) from the FHIR Converter project and customize it.
+2. Compress the templates and push to your ACR. You can use a [commandline tool](https://github.com/microsoft/FHIR-Converter/releases/tag/v3.0) to help upload your templates.
+3. Register the ACR server to your FHIR service configuration by setting `FhirServer:Operations:ConvertData:ContainerRegistryServers:0` to your ACR login server. You can register multiple ACRs by adding more entries in the configuration as the following picture shows.  
+![acr-registration](./images/convert-data/acr-registration.png)
+4. Give your FHIR service the `AcrPull` permission for the ACR you have registered. 
+![acr-rbac](./images/convert-data/acr-rbac.png)
+5. Now you are able to use your custom template collection in the format of `<RegistryServer>/<imageName>:<imageTag>` `<RegistryServer>/<imageName>@<imageDigest>`. 
+Both image tag and digest are supported in the reference string, but we strongly recommend to choose digest because it's immutable and stable. 
+
+## Request format
+The request payload is passed as [Parameters](http://hl7.org/fhir/parameters.html) resource to the ```$convert-data``` api.
+To convert Hl7v2 data to FHIR, you need to pass four parameters:
+1. *inputData*: raw string content of the input data.
+2. *inputDataType*: data type of your input, currently only accepts *Hl7v2*.
+3. *templateCollectionReference*: reference string of your template collection, can be either the built-in templates ```microsofthealth/fhirconverter:default``` or your custom image reference.
+4. *entryPointTemplate*: the entry point template to process (render) the
```

Comment:
>entryPointTemplate [](start = 4, length = 18)

It looks like it got renamed to `rootTemplate`

Reply:
Thanks:)